.github/workflows/tests.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           node-version: ${{ matrix.node-version }}
 
       - name: Install bison app packages
-        run: yarn install
+        run: yarn install --immutable
```

Comment:
Was there a reason we weren't using `--immutable` here? Maybe not needed?

Reply:
The `yarn.lock` for the test bison app is created in the CI so it's not needed but I think this is fine to add here.